tests/jenkins/TestPromoteRepos.groovy

```diff
@@ -43,6 +43,6 @@ class TestPromoteYumRepos extends BuildPipelineTest {
 
     @Test
     public void testDefault() {
-        super.testPipeline("tests/jenkins/jobs/PromoteYumRepos_Jenkinsfile")
+        super.testPipeline("tests/jenkins/jobs/PromoteRepos_Jenkinsfile")
```

Comment:
Can we add actual assertions to this test? Right now it is just emitting a txt file.

Reply:
Example: https://github.com/opensearch-project/opensearch-build-libraries/blob/main/tests/jenkins/TestPublishToArtifactsProdBucket.groovy#L35-L51

Reply:
fixed now.